app/views/active_storage/blobs/_blob.html.erb

```diff
@@ -10,7 +10,7 @@
       <span class="attachment__name"><%= blob.filename %></span>
       <span class="attachment__size"><%= number_to_human_size blob.byte_size %></span>
     <% end %>
-    <%= link_to rails_blob_url(blob, subdomain: Apartment::Tenant.current),  target: '_blank' do %>
+    <%= link_to Rails.application.routes.url_helpers.rails_blob_url(blob, subdomain: Apartment::Tenant.current, host: Rails.application.config.action_mailer.default_url_options[:host]),  target: '_blank' do %>
```

Comment:
@alis-khadka if you're looking for the `host` take a look at using `ENV[APP_HOST]` here instead. Its better than using the action mailer default which is only initialized at boot time (ie; probably wont change & you want that)

Reply:
edit: I was mistaken. Initialization doesnt matter in this context (both the ENV and Rails.application.blah.blah are both boot time initialized)
But its a bit clearer when we use ENV[APP_HOST]:

``` ruby
ubuntu@ip-172-31-68-13:/var/www/violet/current$ rails c
RAILS_ENV=staging environment is not defined in config/webpacker.yml, falling back to production environment
Loading staging environment (Rails 6.1.5)
irb(main):001:0> ENV['APP_HOST']
=> "restarone.solutions"
irb(main):002:0> Rails.application.config.action_mailer.default_url_options[:host]
=> "restarone.solutions"
irb(main):003:0>
```

Reply:
Sure, I will push the update.